addons/account_fleet/views/fleet_vehicle_log_services_views.xml

```diff
@@ -13,10 +13,17 @@
                         type="object"
                         class="oe_stat_button"
                         icon="fa-pencil-square-o"
-                        invisible="not account_move_line_id.move_id">
+                        invisible="not account_move_line_id.move_id or not account_move_line_id">
                         <div class="o_field_widget o_stat_info">
-                            <span class="o_stat_text text-success" invisible="account_move_state != 'posted'" title="Service's Bill"> Service's Bill</span>
-                            <span class="o_stat_text text-warning" invisible="account_move_state != 'draft'" title="Service's Bill"> Service's Bill</span>
+                            <span class="o_stat_text text-success"
+                                invisible="account_move_state != 'posted'"
+                                title="Service's Bill"> Service's Bill</span>
```

Comment:
why this space? same below

Reply:
I kept it, I guess it was added for spacing. It probably can be resolved through the use of margins, I'll check what I can do to remove it :ok_hand: